packages/patternfly-3/patternfly-react-extensions/less/vertical-tabs.less

```diff
@@ -41,4 +37,40 @@
   &:first-of-type {
     margin-top: 0;
   }
+
+  > .vertical-tabs-pf {
+    > .vertical-tabs-pf-tab {
+      position: initial;
+      padding-left: 15px;
+    }
+  }
+}
+
+.vertical-tabs-pf.restrict-tabs {
+  .vertical-tabs-pf-tab {
```

Comment:
(continued...) change `.vertical-tabs-pf-tab` here to `.vertical-tabs-pf`?

That way, when you're hiding a list of tabs, you're hiding the list itself, instead of hiding the list items (which leaves the list visible, even if you can't really see it on the screen).

Reply:
Since the .vertical-tabs-pf-tab is not shown unless it is part of the tree, its child list is not shown unless one of its child items is shown.

Reply:
gotcha, makes sense.